README.md

```diff
@@ -0,0 +1,183 @@
+
+- FontAwesome: A library for adding scalable vector icons to your app.
+
+### Tech Stack <a name="tech-stack"></a>
+- [React](https://react.dev/learn/installation/)
+ - [Redux](https://redux.js.org/)
+
+### Key Features <a name="key-features"></a>
+
+- Real-time Data: Get access to the latest    COVID-19 statistics, including confirmed cases, deaths, and more.
+- Global and Country Data: View global statistics as well as country-specific information.
+- Mobile-Friendly: The app is responsive and mobile-friendly, ensuring a seamless experience on various devices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Link](https://dev--grand-beignet-763e0c.netlify.app/)
+
+## Presentation
+
+[Link to my presentation](https://www.loom.com/share/0e997299fe7a4a4cbf6e1f841ff48830?sid=390bba7e-89f2-465e-9aa2-3a133de8767c)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- An npm installed on your syatem.
+
+#### Windows
+
+[Visit the link to download the package](https://nodejs.org/dist/v18.15.0/node-v18.15.0-x86.msi)
+
+#### Linux
+
+```sh
+ sudo apt install nodejs
+```
+
+#### MacOS
+
+[Visit the link to download the package](https://nodejs.org/dist/v18.15.0/node-v18.15.0.pkg)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/hydaline-code/covid-19Infohub
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Running Tests
+
+To to test the project, checkout [Jest](https://jestjs.io/)
+
+```sh
+npm install jest --save-dev
+```
+
+Then, to start the test:
+
+```sh
+  npm run test
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run dev
+```
+
+### Deployment
+
+The project will be deployed soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Hydaline djoungang**
+- [GitHub](https://github.com/hydaline-code)
+- [LinkedIn](https://www.linkedin.com/in/hydaline-djougang-0851aa21a/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[UI enhancemets is coming soon.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
+- Inspiration
+- Thanks to Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL BUT HIGHLY RECOMMENDED] Please ensure that you fix your License file, the link is broken. :clap: 
- Below is the screenshot. :arrow_down: 
![Mit4004](https://github.com/hydaline-code/covid-19-infohub/assets/43528594/cc835742-b201-4d87-84ef-10afc741bbf6)